src/__tests__/utils/test-utils.ts

```
@@ -0,0 +1,503 @@
+    console.log('Created request:', {
+        url,
+        method,
+        headers: request.headers,
+        body: body ? JSON.stringify(body) : undefined
+    });
+    return request;
+};
+
+// Test API response helper
+export const testApiResponse = async (
+    handler: Function,
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)